wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/RdfConverter.java

```diff
@@ -363,6 +364,24 @@ void writeTermTriples(Resource subject, URI predicate,
 		}
 	}
 
+	/**
+	 * Writes a triple for the {@link StatementRank} of a {@link Statement} to
+	 * the dump.
+	 * 
+	 * @param subject
+	 * @param rank
+	 */
+	void writeStatementRankTriple(Resource subject, StatementRank rank) {
+		String rankRepresentation = rankToRdfRepresentation(rank);
```

Comment:
Rename local variable to match renamed method (see below), or simply inline this call.